### PR TITLE
changelog and version bump for 1.1.2

### PR DESCRIPTION
**Pull request by reaperhulk**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+1.1.2 - 2015-12-10
+~~~~~~~~~~~~~~~~~~
+
+* Fixed a SIGBUS crash with the OS X wheels caused by redefinition of a
+  method.
+* Fixed a runtime error ``undefined symbol EC_GFp_nistp224_method`` that
+  occurred with some OpenSSL installations.
```
Comment (alex) [inline CHANGELOG.rst]: We're also bumping the OpenSSL version on OS X and Windows right?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+1.1.2 - 2015-12-10
+~~~~~~~~~~~~~~~~~~
+
+* Fixed a SIGBUS crash with the OS X wheels caused by redefinition of a
+  method.
+* Fixed a runtime error ``undefined symbol EC_GFp_nistp224_method`` that
```
Comment (alex) [inline CHANGELOG.rst]: Openssl doesn't like the spelling of "runtime"

**Comment by reaperhulk** [comment]
jenkins, retest this please

**Comment by reaperhulk** [comment]
Jenkins, retest this please

**Pull request merged**